recipes/skyr-url/all/conanfile.py

```diff
@@ -47,21 +47,25 @@ def config_options(self):
     def configure(self):
         if self.settings.get_safe("compiler.cppstd"):
             tools.check_min_cppstd(self, self._minimum_cpp_standard)
-        min_version = self._minimum_compilers_version.get(
-            str(self.settings.compiler))
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
         if not min_version:
             self.output.warn("{} recipe lacks information about the {} compiler support.".format(
                 self.name, self.settings.compiler))
         else:
             if tools.Version(self.settings.compiler.version) < min_version:
                 raise ConanInvalidConfiguration("{} requires C++17 support. The current compiler {} {} does not support it.".format(
                     self.name, self.settings.compiler, self.settings.compiler.version))
-                
-        if self.options.with_fs and str(self.settings.compiler) == "apple-clang":
-            raise ConanInvalidConfiguration("apple-clang 11 is currently not support with filesystem")
-            
+
+        if self.options.with_fs and self.settings.compiler == "apple-clang":
+            raise ConanInvalidConfiguration("apple-clang currently does not support with filesystem")
+
         if self.options.shared:
-            raise ConanInvalidConfiguration("shared is currently not supported")
+            # https://github.com/cpp-netlib/url/blob/dd345361ed86e4c1cabfe94743a8e769b346840c/src/CMakeLists.txt#L17
+            raise ConanInvalidConfiguration("shared is currently not supported by upstream")
+
+        if tools.Version(self.version) >= "1.13.0" and not self.settings.compiler == "Visual Studio":
+            # There's tedious compilation errors in C3i against ranges-v3
+            raise ConanInvalidConfiguration("{}/{} with {} is currently not supported".format(self.name, self.version, self.settings.compiler))
```

Comment:
If someone needs other platforms they will sadly have to figure it out themselves

Reply:
Does upstream know about the problem?

Reply:
I don't know what was the issue, but it's perfectly fine with apple-clang. And you should disable `with_fs` by default for apple-clang.

Reply:
Given how much has changed in the last year I am not surprised.